app/src/components/constants.js

```diff
@@ -32,6 +32,12 @@ module.exports = Object.freeze({
     URL: 'url'
   },
 
+  /**
+   * Generic email regex modified to require domain of at least 2 characters
+   * @see {@link https://emailregex.com/}
+   */
+  EMAILREGEX: '^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]{2,})+$',
```

Comment:
Wondering would be better to define the constant like: "EMAIL_REGEX" instead as combined (one word)? In the future if we have a multiple word constant, it might look nicer to split by _